com.unity.formats.fbx/Documentation~/recorder.md

```diff
@@ -2,8 +2,9 @@
 
 With the FBX Exporter and the [Unity Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@2.1/index.html), it is possible to export animations (including [Cinemachine](https://docs.unity3d.com/Packages/com.unity.cinemachine@2.3/manual/index.html) camera animations) directly to FBX in a few easy steps:
```

Comment:
Should this be @2.2 for the recorder?

(Just checked and yes the version number is required)